fearless/Modules/Send/SendPresenter.swift

```diff
@@ -472,51 +481,91 @@ final class SendPresenter {
                 totalBalance: eqUilibriumTotalBalance
             )
         }
-
-        DataValidationRunner(validators: [
-            dataValidatingFactory.has(fee: fee, locale: selectedLocale, onError: { [weak self] in
-                self?.refreshFee(for: chainAsset, address: address)
-            }),
-            dataValidatingFactory.canPayFeeAndAmount(
-                balanceType: balanceType,
-                feeAndTip: (fee ?? 0) + (tip ?? 0),
-                sendAmount: sendAmountDecimal,
-                locale: selectedLocale
-            ),
-            dataValidatingFactory.exsitentialDepositIsNotViolated(
+        var validators: [DataValidating]
+        switch validationCase {
```

Comment:
please explain why you need different validation cases

Reply:
because we need validate existential deposit when amount changed, and validate all cases when we try to go to next screen